index.html

```diff
@@ -1148,6 +1148,31 @@ <h3>canonicalizationAlgorithm</h3>
         </pre>
       </section>
 
+      <section id="sameAs" about="https://w3id.org/security#sameAs"
```

Comment:
Not sure what's going on here.  The property in the context is `owl:sameAs`, not `sec:sameAs`.

Reply:
I suppose this is an artifact of how https://w3id.org/security handles vocabulary definitions.

Because of the way that the VC Data Model, DID Core Spec and DID Spec Registries rely on this CCG repo for terminology definitions.... I am attempting to define the way sameAs is used in Linked Data Security vocabulary, so that everything is consistent.

If I don't than the behavior of JSON-LD processing software, such as vc.js and jsonld-signatures will be to ignore these properties... 

Can you make a proposed change to ensure that such a thing would not happen?